src/sentry_util.h

```diff
@@ -1,16 +1,45 @@
 #ifndef SENTRY_UTIL_H
 #define SENTRY_UTIL_H
 
+#include "sentry_options.h"
+
 #include <sentry.h>
 #include <godot_cpp/variant/char_string.hpp>
+#include <godot_cpp/variant/packed_string_array.hpp>
 #include <godot_cpp/variant/variant.hpp>
+#include <iostream>
+
+using namespace godot;
 
 class SentryUtil {
 public:
-	static sentry_value_t variant_to_sentry_value(const godot::Variant &p_variant);
-	static godot::CharString get_screen_orientation_cstring(int32_t p_screen = -1);
+	static sentry_value_t variant_to_sentry_value(const Variant &p_variant);
+	static CharString get_screen_orientation_cstring(int32_t p_screen = -1);
 	static void sentry_event_set_context(sentry_value_t p_event, const char *p_context_name, sentry_value_t p_context);
 	static godot::CharString generate_uuid();
+	static sentry_value_t strings_to_sentry_list(const PackedStringArray &p_strings);
+
+	template <typename... Args>
+	static void print_debug(const Variant &p_arg1, const Args &...p_args) {
+		if (SentryOptions::get_singleton()->is_debug_enabled()) {
+			std::cout << "[sentry] DEBUG GODOTSDK ";
```

Comment:
I'd leave out the 'godotsdk' part
```suggestion
			std::cout << "[sentry] DEBUG ";
```

Reply:
Done